oci/spec_opts.go

```diff
@@ -943,6 +943,11 @@ func WithCapabilities(caps []string) SpecOpts {
 		s.Process.Capabilities.Bounding = caps
 		s.Process.Capabilities.Effective = caps
 		s.Process.Capabilities.Permitted = caps
+		if caps == nil {
```

Comment:
```suggestion
		if len(caps) == 0 {
```

I see that `filterCaps` does effectively drop everything if `len(caps)==0` anyway, but it looks a little more explicit in `WithCapabilities` this way.